swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
+info:
+  title: "Tweet API"
+  description: "This provides an API for creating a message (post), reading a message (get), editing a message (put), delete a message (delete)."
+  version: "1.0"
+
+host: localhost
+
+tags:
+- name: "tweets"
+
+schemes:
+  - "https"
+
+consumes:
+  - application/json
+produces:
+  - application/json
+
+paths:
+
+  /tweets:
+  
+    get:
+      tags:
+      - tweets
+      description: "search for a user's tweets"
+      produces:
+      - application/json
```

Comment:
Is there any different output format this API will produce? Since you've defined the output `produces` at the top-level, it's not necessary to repeat `produces` in every endpoint if there is no difference.